.buildkite/pipeline.yml

```diff
@@ -0,0 +1,34 @@
+---
+steps:
+  - type: "waiter"
+
+  - name: ":docker:"
+    command: "bin/buildkite bin/ci build"
+    agents:
+      queue: native
+      docker: builder
+
+  - type: "waiter"
+
+  - name: ":pill:"
+    command: "bin/buildkite bin/ci run npm test"
+    agents:
+      queue: elastic
+
+  - type: "waiter"
+
+  - name: ":rocket: Publish"
+    type: "manual"
+    branches: "master"
+
+  - name: ":speech_balloon:"
+    command: "bin/buildkite bin/ci announce releasing-production"
+    branches: "master"
+    agents:
+      queue: elastic
+
+  - name: ":speech_balloon: Publish Treehorn"
+    command: "bin/buildkite bin/ci create-deploy-candidate production"
+    branches: "master"
+    agents:
+      queue: elastic
```

Comment:
You don't wanna celebrate with a "Released production!" https://github.com/everydayhero/plain-pipeline/blob/master/template/.buildkite/pipeline.yml#L91 (it might actually be the case that given the new slack of updating a messages colour before, during and after that it's required to make it green?)